crud/src/main/resources/json-schema/crudCommon.json

```diff
@@ -55,17 +55,8 @@
         },
         "execution": {
             "type": "object",
-            "properties": {
-                "timeLimit": {
-                    "type": "integer",
-                    "description": "The upper time limit for the call in milliseconds. If the call does not complete before the limit expires, call fails with partial results written. The return status will contain what items are written."
-                },
-                "asynchronous": {
-                    "type": "integer",
-                    "description": "Given as a time limit in milliseconds. Once that limit is exceeded, the call returns, but execution continues. The return value will contain a task handle that can be used to check for execution status."
-                }
-            },
-            "additionalProperties": false
+            "description": "Execution options captured as key/value pairs.  See documentation on http://lightblue.io for what is supporeted",
+            "additionalProperties": true
```

Comment:
Last thing on my to-do list for adding the maxQueryTimeMS is to update user documentation.  My thinking was to just deal with this in one place, in the docs.  Especially now that execution options are passed on the operational context, there is no code (none) in core that will change if a new option is supported by a controller implementation.  It is therefore not desirable, in my opinion, to manage a concrete set of properties here if that means we do have to update the schema for those options that are controller specific.